code/src/components/ThoughtList.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+import Thought from './Thought'
+import '../index.css'
+
+const ThoughtList = ({ messageList, handleLikesIncrease }) => {
+  return (
+    <>
+      {messageList.map((message) => (
```

Comment:
Your code is just so neat and well done! And on top of that you are a great teammate! You will do great things Anna-Kaisa! 🤩  🎉